### DISCUSSION TITLE
Thumbnails empty

### DISCUSSION THREAD

**Discussion by foxbw** [LycheeOrg/Lychee#2509] [Q&A] [answered]
Hello,

I have installed Lychee successfully. Then I want to upload some photos. There will be no error. If I want to display the photos the thumbnails will be empty.

![image001](https://github.com/user-attachments/assets/f6029acd-966b-4bc1-b032-61e7966f71c5)

Can somebody help me? Do you need further information?

**Comment by ildyria** [comment]
Can you paste the content of your diagnostic page?

**Reply by foxbw** [reply]
This is the result of the diagnostic page:

    Diagnostics
    -------
    Warning: Dropbox import not working. dropbox_key is empty.
    Warning: You may experience problems when uploading a photo of large size or handling many/large albums. Take a look in the FAQ for details.
    Info: Latest version of PHP is 8.3




    System Information
    --------------
    Lychee Version (git):                    master (29b5867) -- Up to date (-0.00099765815867866 years ago).
    DB Version:                              5.5.1
    
    composer install:                        --no-dev
    APP_ENV:                                 production
    APP_DEBUG:                               false
    APP_URL:                                 set
    APP_DIR:                                 set
    LOG_VIEWER_ENABLED:                      true
    LIVEWIRE_ENABLED:                        false
    
    System:                                  Linux
    PHP Version:                             8.2.21
    PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
    Timezone:                                Europe/Paris
    Max uploaded file size:                  512M
    Max post size:                           512M
    Livewire chunk size:                     12.00 MB
    Max execution time:                      30
    MySQL Version:                           10.6.18-MariaDB-0ubuntu0.22.04.1
    
    exec() Available:                        yes
    Imagick Available:                       1
    Imagick Enabled:                         1
    Imagick Version:                         1691
    GD Version:                              2.3.3
    Number of foreign key:                   12 found.

    Config Information
    --------------
    version:                                 050501
    check_for_updates:                       0
    sorting_photos_col:                      taken_at
    sorting_photos_order:                    ASC
    sorting_albums_col:                      max_taken_at
    sorting_albums_order:                    ASC
    imagick:                                 1
    skip_duplicates:                         1
    small_max_width:                         0
    small_max_height:                        360
    medium_max_width:                        1920
    medium_max_height:                       1080
    lang:                                    de
    image_overlay_type:                      desc
    default_license:                         reserved
    compression_quality:                     90
    grants_full_photo_access:                1
    delete_imported:                         0
    mod_frame_enabled:                       1
    mod_frame_refresh:                       30
    thumb_2x:                                1
    small_2x:                                1
    medium_2x:                               1
    landing_page_enable:                     0
    force_migration_in_production:           0
    search_public:                           0
    hide_version_number:                     0
    SL_enable:                               0
    SL_for_admin:                            0
    recent_age:                              1
    grants_download:                         0
    photos_wraparound:                       1
    map_display:                             1
    zip64:                                   1
    map_display_public:                      1
    map_provider:                            Wikimedia
    force_32bit_ids:                         0
    map_include_subalbums:                   1
    update_check_every_days:                 3
    has_exiftool:                            0
    share_button_visible:                    0
    import_via_symlink:                      0
    has_ffmpeg:                              0
    location_decoding:                       1
    location_decoding_timeout:               30
    location_show:                           1
    location_show_public:                    1
    rss_enable:                              0
    rss_recent_days:                         7
    rss_max_items:                           100
    prefer_available_xmp_metadata:           0
    editor_enabled:                          1
    lossless_optimization:                   0
    swipe_tolerance_x:                       150
    swipe_tolerance_y:                       250
    local_takestamp_video_formats:           .avi|.mov
    log_max_num_line:                        1000
    unlock_password_photos_with_url_param:   0
    nsfw_visible:                            0
    nsfw_blur:                               0
    nsfw_warning:                            0
    nsfw_warning_admin:                      0
    nsfw_banner_override:                    
    map_display_direction:                   1
    album_subtitle_type:                     oldstyle
    upload_processing_limit:                 4
    new_photos_notification:                 1
    legacy_id_redirection:                   1
    zip_deflate_level:                       6
    default_album_protection:                1
    album_decoration:                        layers
    album_decoration_orientation:            row-reverse
    allow_username_change:                   1
    auto_fix_orientation:                    1
    use_job_queues:                          0
    random_album_id:                         starred
    use_last_modified_date_when_no_exif_date: 0
    layout:                                  justified
    date_format_photo_thumb:                 M j, Y, g:i:s A e
    date_format_photo_overlay:               M j, Y, g:i:s A e
    date_format_sidebar_uploaded:            M j, Y, g:i:s A e
    date_format_sidebar_taken_at:            M j, Y, g:i:s A e
    date_format_hero_min_max:                F Y
    date_format_hero_created_at:             M j, Y, g:i:s A T
    date_format_album_thumb:                 M Y
    upload_chunk_size:                       0
    nsfw_banner_blur_backdrop:               0
    search_pagination_limit:                 1000
    search_minimum_length_required:          4
    photo_layout_justified_row_height:       320
    photo_layout_masonry_column_width:       300
    photo_layout_grid_column_width:          250
    photo_layout_square_column_width:        200
    photo_layout_gap:                        12
    display_thumb_album_overlay:             always
    display_thumb_photo_overlay:             hover
    default_album_thumb_aspect_ratio:        1/1
    use_album_compact_header:                0
    login_button_position:                   left
    back_button_enabled:                     1
    back_button_text:                        Return to Home
    current_job_processing_visible:          1
    SA_random_thumbs:                        0
    job_processing_queue_visible:            1
    login_required:                          0
    enable_unsorted:                         1
    enable_starred:                          1
    enable_recent:                           1
    enable_on_this_day:                      1
    login_required_root_only:                1

**Reply by ildyria** [reply]
Okay, can you check the console of your web-browser, see if there are any errors there?

**Reply by foxbw** [reply]
When I open the album "unsorted" there are errors and a lot of warnings in the web-browser console

![grafik](https://github.com/user-attachments/assets/499d1194-862a-4bb5-9486-b9c891d2dc9d)

![grafik](https://github.com/user-attachments/assets/430d39ff-e613-4830-89be-bb9173c20367)

When I open the Log-Viewer a new tab in the browser will open but there is nothing to see. In the browser console the following error appears:

![grafik](https://github.com/user-attachments/assets/1bd529c0-8b5c-462e-a671-b804e6a03edd)

**Reply by ildyria** [reply]
what is the details in the GET of an image ?

**Reply by foxbw** [reply]
![grafik](https://github.com/user-attachments/assets/59f2174a-a093-49ca-8b22-d44a39c5f71c)

**Reply by ildyria** [reply]
So that's the problem.
Can you check in your upload/small/92/08 folder if the image is indeed here?

**Reply by foxbw** [reply]
Yes it is there.

![grafik](https://github.com/user-attachments/assets/6a0f70f5-b0e6-40e9-82fe-6e5296a01f55)

**Reply by ildyria** [reply]
Then the URL provided for the picture is wrong.
What did you set as APP_DIR ?

**Reply by foxbw** [reply]
![grafik](https://github.com/user-attachments/assets/6a5bf23d-9d27-4201-b947-e3df372a0c90)

**Reply by ildyria** [reply]
remove it. :)

**Reply by foxbw** [reply]
![grafik](https://github.com/user-attachments/assets/9cbf8f7b-8697-4d20-b5af-8234b3e93be0)

Now it is the wrong adress for the picture

![grafik](https://github.com/user-attachments/assets/aacefc76-bb95-4e42-a63a-6c53b04fcaf3)

**Reply by foxbw** [reply, marked as answer]
As I just now saw, the address for the picture in my post above is also wrong. “public” is missing there.

**Reply by ildyria** [reply]
Yep, even though it should not be required because if APP_DIR is not defined at all this should be set automatically. :)

**Reply by ildyria** [reply]
But I guess because you are hosting locally behind path you need it.
I would recommend to set up your Apache and a local host name so that you don't need it otherwise your .env might be exposed. :)

**Reply by foxbw** [reply]
Thank you!
I set the right directory and now it is working.
